openmrs-19/src/main/resources/velocity.templates/task-data-provider.vm

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
All right, the configs are added to the display names, but are they used later on during retrieval of the data...? It seems like all those generated lookups will still do the same thing.